build.gradle.kts

```
@@ -0,0 +1,12 @@
+plugins {
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "1.0.6"
```

_suggestion_
A newer version seems to be available:
```suggestion
  id("uk.gov.justice.hmpps.gradle-spring-boot") version "1.1.1"
```